.gitignore

```diff
@@ -21,7 +21,7 @@ parts/
 sdist/
 var/
 node_modules/
-webpack-stats.json/
+webpack-stats.json
```

Comment:
@VKotsovskyi, this slash was added by mistake in previous pull request. It should not be there.